lib/server.js

```diff
@@ -191,7 +191,7 @@ Server.prototype.close = function () {
   if (this.ws) {
     debug('closing webSocketServer');
     this.ws.close();
-    delete this.ws;
+    // don't delete this.ws because it can be used again if the server starts listening again
```

Comment:
At the end of the day, I think that, since deleting `this.ws` is such a tempting thing to do, the comment might help future contributors understand why we can't do it, and will save them time it would take them to figure it out on their own.